plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/EclipseWorkspaceImpl.java

```diff
@@ -130,6 +131,18 @@ protected void loadWorkspaceProjects() throws DBException {
                 if (activeProject == null || (!CommonUtils.isEmpty(activeProjectName) && project.getName().equals(activeProjectName))) {
                     activeProject = projectMetadata;
                 }
+
+                // To avoid search/replace functionality accidentally corrupting workspace configuration we need to
+                // mark metadata folder as hidden see dbeaver/dbeaver#20759
```

Comment:
```suggestion
                // To avoid accidental corruption of the workspace configuration by search/replace commands,
                // we need to mark metadata folder as hidden (see dbeaver/dbeaver#20759)
```